code/_onclick/mutual_cuff.dm

```diff
@@ -93,21 +93,17 @@
 		handcuffed_to.u_equip(src)
 
 /mob/living/carbon/proc/on_mutual_cuffed_move(atom/movable/mover)
-	if (!isturf(loc) || (mutual_handcuffed_to && get_dist(mutual_handcuffed_to, src) > 3)) // We moved to a mech, a sleeper, or a locker, or got teleported
+	if (!isturf(loc) || (mutual_handcuffed_to && get_dist(mutual_handcuffed_to, src) > 3) || locked_to) // We moved to a mech, a sleeper, or a locker, was teleported, or tried pulling someone who was locked in place
 		var/obj/item/weapon/handcuffs/H = mutual_handcuffs
 		if (!istype(H))
 			return
-		H.visible_message("<span class='warning'>\The [H] breaks!</span>")
+		H.visible_message("<span class='warning'>\The [H] break!</span>")
```

Comment:
Did a quick skim and currently all obj/item/weapon/handcuffs are plural, so this change works! The way that checking for plurality is crazy in certain areas of the code (see stacks for example, tooths vs teeth).

Simple future proofing could be a check on the last letter to see if it's an s, and change breaks to break accordingly.